cmds/exp/tcpdump/gen.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2024 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build ignore
+
+package main
+
+import (
+	"bufio"
+	"io/ioutil"
+	"log"
+	"os"
+	"regexp"
+	"strings"
+)
+
+func main() {
+	file, err := os.Open("/etc/services")
+	if err != nil {
+		log.Fatalf("Error opening /etc/services: %v", err)
+	}
+	defer file.Close()
+
+	re := regexp.MustCompile(`^(\S+)\s+(\d+)/\S+`)
+
+	ports := make(map[string]string)
+
+	scanner := bufio.NewScanner(file)
```

Comment:
you can do this BUT: you know that the file will be tiny. Do you can use os.ReadFile to read it in, strings.Split on \n, 
lines = strings.Split(data, "\n")
and then your for loop will be for _, line := range lines

It's a bit less complex than dealing with scanner. But this code is fine. Thanks for using /etc/servies.

Reply:
Good suggestion. Changed :+1: